File: app/src/main/java/com/dhsdevelopments/potato/clientapi/unreadnotification/UpdateUnreadNotificationRequest.java
package com.dhsdevelopments.potato.clientapi.unreadnotification;

import com.google.gson.annotations.SerializedName;

public class UpdateUnreadNotificationRequest
{
    @SerializedName( "token" )
    public String token;

    @SerializedName( "add" )
    public boolean add;

    public UpdateUnreadNotificationRequest( String token, boolean add ) {
        this.token = token;
        this.add = add;
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/unreadnotification/UpdateUnreadNotificationRequest.kt
package com.dhsdevelopments.potato.clientapi.unreadnotification

import com.google.gson.annotations.SerializedName

@Suppress("unused")
class UpdateUnreadNotificationRequest(
        @SerializedName("token")
        var token: String,

        @SerializedName("add")
        var add: Boolean)
